tests/test_integration.py

```diff
@@ -15,20 +15,36 @@ class TestNotify:
         }
     }
 
+    def set_email(self, email):
+        payload_copy = self.payload_template.copy()
+        payload_copy["payload"]["fulfilmentRequest"]["email_address"] = email
+        return payload_copy
+
+    # The following tests will not work as the Notify API key has been removed until a mock service can be introduced
```

Comment:
It is probably worth creating a ticket on the backlog for this, although depending on whether we ever need emails/gov notify again it might hang around there for a while 😅

Reply:
Ticket added [here].(https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=2213&projectKey=ECI&view=detail&selectedIssue=ECI-1376). Let me know if theres anything I need to change